Scripts/Services/NaturalLanguageUnderstanding/V1/NaturalLanguageUnderstandingService.cs

```diff
@@ -1424,18 +1424,18 @@ private void OnCreateClassificationsModelResponse(RESTConnector.Request req, RES
         /// <summary>
         /// List classifications models.
         ///
-        /// (Beta) Returns all custom classifications models associated with this service instance.
+        /// Returns all custom classifications models associated with this service instance.
         /// </summary>
         /// <param name="callback">The callback function that is invoked when the operation completes.</param>
-        /// <returns><see cref="ListClassificationsModelsResponse" />ListClassificationsModelsResponse</returns>
-        public bool ListClassificationsModels(Callback<ListClassificationsModelsResponse> callback)
+        /// <returns><see cref="ClassificationsModelList" />ClassificationsModelList</returns>
+        public bool ListClassificationsModels(Callback<ClassificationsModelList> callback)
```

Comment:
is this breaking? response model changed names

Reply:
that's the fix for the NLU issue. This function was not working before